services/task-svc/internal/patient/models/models.go

```diff
@@ -13,9 +12,9 @@ type Base struct {
 
 type Patient struct {
 	Base
-	ID             uuid.UUID             `gorm:"column:id;default:uuid_generate_v4()"`
-	OrganizationID uuid.UUID             `gorm:"column:organization_id"`
-	BedID          *uuid.UUID            `gorm:"column:bed_id;default:NULL"`
-	Tasks          []taskModels.Task     `gorm:"foreignKey:PatientId"`
-	IsDischarged   soft_delete.DeletedAt `gorm:"column:is_discharged;softDelete:flag;default:0"`
+	ID             uuid.UUID         `gorm:"column:id;default:uuid_generate_v4()"`
+	OrganizationID uuid.UUID         `gorm:"column:organization_id"`
+	BedID          *uuid.UUID        `gorm:"column:bed_id;default:NULL"`
+	Tasks          []taskModels.Task `gorm:"foreignKey:PatientId"`
+	IsDischarged   int               `gorm:"column:is_discharged;softDelete:flag;default:0"`
```

Comment:
Why did you use an int over a boolean? Rather make it a bool or enum, if the gorm soft_delete doesn't work and make sure that all gets and updates are still returning only not discharged patients
@MaxSchaefer do you have a good solution to this?

Reply:
Otherwise I have problems creating a patient

Reply:
thats probably because of the soft delete flag of gorm, but I am unsure how to properly resolve this. Most likely removing the softdelete in the gorm annotation, but that would require several other changes

Reply:
or better use the unscope of gorm to find the soft deleted (discharged) patients see the [documentation](https://gorm.io/docs/delete.html#Find-soft-deleted-records)

Reply:
@florianjs I am not quite sure what problems you are having when creating patients with `soft_delete.DeletedAt`. Maybe you can elaborate this in https://github.com/helpwave/services/issues/329.